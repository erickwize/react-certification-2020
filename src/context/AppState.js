/* global gapi*/
/* eslint-disable react/prop-types */
import React, { useReducer } from "react";
import AppContext from "./appContext";
import AppReducer from "./appReducer";
import validateItems from "../utils/validateItems";
import initGapi from "../utils/initGapi";
import { GET_RESULT_VIDEOS, GET_RELATED_VIDEOS, SET_LOADING } from "./types";

const AppState = (props) => {
  const initialState = {
    searchText: "Moonwalk",
    resultVideos: [],
    selectedVideo: {},
    relatedVideos: [],
    showPlayer: false,
    loading: true,
  };

  const [state, dispatch] = useReducer(AppReducer, initialState);

  // HANDLE INIT GAPI
  const handleInitGapi = async () => {
    initGapi();
    setTimeout(() => {
      getResultVideos(state.searchText);
    }, 1000);
  };

  // GET RESULT VIDEOS
  const getResultVideos = async (query) => {
    setLoading();
    let items;

    try {
      const response = await gapi.client.youtube.search.list({
        part: ["snippet"],
        maxResults: 50,
        q: query,
        type: ["video"],
      });
      items = validateItems(response.result.items);
    } catch (error) {
      console.error("getResultVideos: Something went wrong... ", error);
    }

    dispatch({
      type: GET_RESULT_VIDEOS,
      payload: { query, items },
    });
  };

  // GET RELATED VIDEOS
  const getRelatedVideos = async (video) => {
    setLoading();
    let items;

    try {
      const response = await gapi.client.youtube.search.list({
        part: ["snippet"],
        maxResults: 50,
        type: ["video"],
        relatedToVideoId: video.id,
      });
      items = validateItems(response.result.items);
    } catch (error) {
      console.error("getRelatedVideos: Something went wrong... ", error);
    }

    dispatch({
      type: GET_RELATED_VIDEOS,
      payload: { video, items },
    });
  };

  // SET LOADING
  const setLoading = () => {
    dispatch({ type: SET_LOADING });
  };

  return (
    <AppContext.Provider
      value={{
        searchText: state.searchText,
        resultVideos: state.resultVideos,
        selectedVideo: state.selectedVideo,
        relatedVideos: state.relatedVideos,
        showPlayer: state.showPlayer,
        loading: state.loading,
        handleInitGapi,
        getResultVideos,
        getRelatedVideos,
        setLoading,
      }}
    >
      {props.children}
    </AppContext.Provider>
  );
};

export default AppState;
