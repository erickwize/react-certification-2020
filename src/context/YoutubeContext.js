import React, { createContext, useState } from 'react'
import { saveArrayOnLocalStorage } from '../utils/localStorage'

const API_KEY = process.env.REACT_APP_YT_API_KEY
const MAX_RESULTS = 20

const YoutubeContext = createContext()

const SEARCH_URL = `https://www.googleapis.com/youtube/v3/search?part=snippet&key=${API_KEY}&type=video&maxResults=${MAX_RESULTS}&`

function YoutubeContextProvider({ children }) {
	const [videos, setVideos] = useState([])
	const defaultContext = {
		videos,
		search: term => {
			fetch(`${SEARCH_URL}q=${term}`)
				.then(response => response.json())
				.then(data => {
					if (data?.items?.length) {
						setVideos(data.items)
						saveArrayOnLocalStorage('videos', data.items)
					} else setVideos([])
				})
				.catch(error => {
					console.error(error)
				})
		},
		setVideosContext: newVideos => {
			setVideos(newVideos)
		},
	}

	return (
		<YoutubeContext.Provider value={defaultContext}>{children}</YoutubeContext.Provider>
	)
}

export { YoutubeContext, YoutubeContextProvider }
