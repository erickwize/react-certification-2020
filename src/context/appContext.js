import { createContext, useContext } from "react";

const appContext = createContext();

export const useAppContext = () => {
  const context = useContext(appContext);
  if (context === undefined) {
    throw new Error(
      "useAppContext must be used within AppState component tree"
    );
  }
  return context;
};

export default appContext;
