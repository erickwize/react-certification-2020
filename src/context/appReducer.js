import { GET_RESULT_VIDEOS, GET_RELATED_VIDEOS, SET_LOADING } from "./types";

export default (state, action) => {
  switch (action.type) {
    case GET_RESULT_VIDEOS:
      return {
        ...state,
        searchText: action.payload.query,
        resultVideos: action.payload.items,
        showPlayer: false,
        loading: false,
      };
    case GET_RELATED_VIDEOS:
      return {
        ...state,
        selectedVideo: action.payload.video,
        relatedVideos: action.payload.items,
        showPlayer: true,
        loading: false,
      };
    case SET_LOADING:
      return {
        ...state,
        loading: true,
      };
    default:
      return state;
  }
};
