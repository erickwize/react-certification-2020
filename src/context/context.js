// import Dexie from 'dexie'
import React, { useEffect, createContext, useReducer, memo } from 'react'
import { useFetch } from '../utils/hooks/useFetch'

export const Context = createContext(null)

export const ContextReducer = (state, action) => {
  switch (action.type) {
    case 'TOGGLE_THEME':
      return {
        ...state,
        theme: !state.theme,
      }
    case 'OPEN_MENU':
      return {
        ...state,
        menu: action.payload,
      }
    case 'GET_DATA':
      return {
        ...state,
        data: action.payload,
      }
    case 'SET_VIEW':
      return {
        ...state,
        view: action.payload,
      }
    case 'SET_TARGET':
      return {
        ...state,
        target: action.payload,
      }
    default:
      return state
  }
}

export const initialState = {
  target: 'Wizeline',
  theme: false,
  menu: false,
  view: 'home',
  data: [],
}
export const ContextProvider = ({ children }) => {
  const [state, dispatch] = useReducer(ContextReducer, initialState)
  const { data, isLoading } = useFetch(state.target)

  useEffect(() => {
    if (data) dispatch({ type: 'GET_DATA', payload: data })
  }, [data])

  return isLoading ? (
    <div> Loading...</div>
  ) : (
    <Context.Provider value={{ state, dispatch }}> {children} </Context.Provider>
  )
}
export default memo(ContextProvider)
