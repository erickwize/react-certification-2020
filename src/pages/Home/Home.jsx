import React, { useRef } from 'react';

import './Home.css';
import VideoList from '../../components/VideoList/VideoList';

function Home() {
  const sectionRef = useRef(null);

  return (
    <div ref={sectionRef}>
      <h2 className="title">Welcome to the Challenge!</h2>
      <VideoList />
    </div>
  );
}

export default Home;
