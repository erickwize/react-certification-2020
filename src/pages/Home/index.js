import React from 'react'
import styled from 'styled-components'
import VideosList from '../../components/VideosList'

const HomeStyled = styled.div`
	padding: 20px;
`
const Title = styled.h1`
	text-align: center;
`

const Home = () => {
	return (
		<HomeStyled>
			<Title>Results</Title>
			<VideosList />
		</HomeStyled>
	)
}

export default Home
