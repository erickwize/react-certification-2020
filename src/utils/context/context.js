import { createContext } from 'react';

export const Context = createContext({
  queryToSearch: null,
  setQueryFromSearchInput: null,
});
