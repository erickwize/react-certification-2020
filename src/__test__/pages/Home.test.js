import React from 'react'
import { render } from '@testing-library/react'
import { YoutubeContextProvider } from '../../context/YoutubeContext'
import Home from '../../pages/Home'

describe('Home', () => {
	it('Home Snapshot', () => {
		const component = render(
			<YoutubeContextProvider>
				<Home />
			</YoutubeContextProvider>
		)
		expect(component.container).toMatchSnapshot()
	})
})
