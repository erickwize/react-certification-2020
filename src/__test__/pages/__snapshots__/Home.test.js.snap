// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home testing Home Snapshot 1`] = `
<div>
  <div
    class="sc-gKsewC giXVud"
  >
    <h1
      class="sc-iBPRYJ duUkbM"
    >
      Videos List
    </h1>
    <div
      class="sc-jSgupP iCxkCv"
    >
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://yt3.ggpht.com/ytc/AAUvwnighSReQlmHl_S_vSfvnWBAG5Cw4A0YxtE0tm5OpQ=s88-c-k-c0xffffffff-no-rj-mo"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Wizeline transforms how teams build technology. Its customers accelerate the delivery of innovative products with proven solutions, which combine Wizeline's ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/nmXMgqjQzls/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Video Tour | Welcome to Wizeline Guadalajara
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Follow Hector Padilla, Wizeline Director of Engineering, for a lively tour of our office. In 2018, Wizeline opened its stunning new office in Guadalajara, Jalisco, ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/HYyRZiwBWc8/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline Guadalajara | Bringing Silicon Valley to Mexico
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Wizeline continues to offer a Silicon Valley culture in burgeoning innovation hubs like Mexico and Vietnam. In 2018, our Guadalajara team moved into a ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/Po3VwR_NNGk/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline hace sentir a empleados como en casa
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            En el 2014, Bismarck fundó Wizeline, compañía tecnológica que trabaja con los corporativos ofreciendo una plataforma para que desarrollen software de forma ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/7PtYNO6g7eI/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            We Are Wizeline
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Engineering a better tomorrow. Wizeline is a global software development company that helps its clients solve their biggest challenges with design and ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/YuW0CE_8i1I/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline Thrives in Mexico City
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Our vibrant Mexico City office is home to agile software engineers, talented UX designers, and brilliant data scientists. Learn about the rich history of Mexico City.
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/CHzlSGRvWPs/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            El plan de Wizeline, una empresa de inteligencia artificial, para ayudar a crecer la comunidad de ciencia de datos en CDMX y todo el país, a través de cursos ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/cjO2fJy8asM/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            A Day in the Life of an Engineering Manager at Wizeline
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Fernando Espinoza shares his experience working as an engineering manager at Wizeline and mentoring other engineers. Learn about Fernando's passions ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/zClI9OjgKXM/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline Technical Writing Academy | Featuring Eduardo Ocejo
          </span>
          <p
            class="sc-eCssSg gCjviD"
          />
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/8bz9R61oY5o/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Silicon Valley en México
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Empresas de Silicon Valley buscan establecerse en México por el gran talento que hay en nuestro país. Es una investigación de Roberto Domínguez.
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/7dJFraOqcoQ/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Why Wizeline? featuring Juan Pablo Villa in Mexico City
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Juan Pablo, known as Gianpa at Wizeline, is a software engineer in our Mexico City office. Gianpa focuses on Android apps, is an integral part of our culture, ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/w-Qwc_XJrWc/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline&#39;s 2020 Year in Review
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            There's no doubt that 2020 has been an unprecedented year. However, amidst all the chaos, we achieved remarkable growth in various areas of our business.
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/rjir_cHTl5w/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Why Wizeline? featuring Hugo Lopez in Mexico City
          </span>
          <p
            class="sc-eCssSg gCjviD"
          />
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/DcFK1x3NHGY/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Why Wizeline? (We&#39;re Hiring in Mexico!)
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            A quick look at why people join Wizeline, what motivates us as a team and what it's like to work in our Guadalajara office. Learn more and apply here: ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/3BzYWAqZgFw/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Why Wizeline? featuring Oswaldo Herrera in Mexico City
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Oswaldo is a software engineering in Wizeline's Mexico City office. He joined Wizeline because of the camaraderie and deep sense of commitment of our teams.
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/3KVFmT-Tp2w/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Caso de Éxito AWS: Wizeline [Spanish]
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Central de socios de APN - https://amzn.to/2S7tIXM Fundada en 2014, Wizeline es una compañía joven e innovadora que nació en la nube para ofrecer soporte ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/aKuPmY2m1Ro/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline&#39;s 2019 Year in Review
          </span>
          <p
            class="sc-eCssSg gCjviD"
          />
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/24sTHUyWhRM/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            [1 of 2] Wizeline CEO shares career lessons from Google
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Founder & CEO Bismarck Lepe on growth opportunities at Wizeline and his career-path experience as an early Google employee. Join our team!
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/IxGc1gSqB3A/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline Data Engineering featuring Tania Reyes
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Tania discovered her interest in Big Data while working at Wizeline and took Wizeline Academy courses to skill up and join the data team. Now, she works on ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/NP1gAnbeNno/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Wizeline Querétaro | Mexico&#39;s New Knowledge Economy (We&#39;re hiring!)
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            A small but mighty (and growing) team, the Queretaro crew has taken ownership of growing the office and brand, speaking at university events, hosting tech ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/F6Krwu6lUc8/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Miriam Godinez | Women in Leadership at Wizeline
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Science and technology always caught Miriam's attention. One of her ultimate goals as a Senior Engineer Manager and Lead from the Mobile Team at Wizeline ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/RFq7gfvhtCk/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Welcome Back to Wizeline Vietnam | Extended Version
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Thanks to swift government action, the COVID-19 situation in Vietnam has reached a point where businesses are able to return to work and reopen offices.
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/E1Vq_A3WKK8/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            How does Wizeline work?
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Wizeline builds teams with a mix of technical and non-technical talent to deliver better products, faster. Learn more about our consulting services: ...
          </p>
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/ZmkslANDz0Q/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            12 Wishes from Wizeline | Happy Holidays 2019
          </span>
          <p
            class="sc-eCssSg gCjviD"
          />
        </div>
      </div>
      <div
        class="sc-bdfBwQ kkuHub"
      >
        <img
          alt="thumbnail"
          class="sc-gsTCUz bnntAB"
          src="https://i.ytimg.com/vi/Nss3EmTDD3s/default.jpg"
        />
        <div
          class="sc-dlfnbm ffjVNP"
        >
          <span
            class="sc-hKgILt gJCItH"
          >
            Why Wizeline?
          </span>
          <p
            class="sc-eCssSg gCjviD"
          >
            Hear from our employees directly about what excites them about their roles here at Wizeline. Wizeline wants to hire the best and the brightest to accelerate their ...
          </p>
        </div>
      </div>
      )
    </div>
  </div>
</div>
`;
